### PR TITLE
libcec 3.0.0

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: Library/Formula/libcec.rb [hunk 1]
```diff
@@ -11,17 +11,31 @@ class Libcec < Formula
     sha1 "5b04eb4b157c34ab543082a9c9c0868450412c37" => :mountain_lion
   end
 
+  resource "platform" do
+    url "https://github.com/Pulse-Eight/platform/archive/1.0.9.tar.gz"
+    sha256 "276dec2b8653fcf9bf636ac929bb014cc0963ccacd67179705d39301a9533b1e"
+  end
+
   depends_on "autoconf" => :build
   depends_on "automake" => :build
```
Comment (mikemcquaid) [inline Library/Formula/libcec.rb]: Are you sure these are still needed? If it's using `cmake` now it seems unlikely. Perhaps `cmake` is the only build-time requirement needed now?

**Comment by DomT4** [comment]
Failed heavily on ML. Usually when ML and ML only fails it's `cxx11` related.

**Comment by nijikon** [comment]
Oh yes! Working! :)

File: Library/Formula/libcec.rb [hunk 2]
```diff
@@ -11,17 +11,36 @@ class Libcec < Formula
...
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
-  depends_on "pkg-config" => :build
+  resource "platform" do
+    url "https://github.com/Pulse-Eight/platform/archive/1.0.10.tar.gz"
+    sha256 "6ba3239cb1c0a5341efcf9488f4d3dfad8c26d6b2994b2b2247e5a61568ab5cd"
+  end
+
+  depends_on "cmake" => :build
```
Comment (DomT4) [inline Library/Formula/libcec.rb]: You'll need `needs :cxx11` one blank line underneath this as well.

File: Library/Formula/libcec.rb [hunk 3]
```diff
@@ -11,17 +11,36 @@ class Libcec < Formula
...
+    sha256 "93ae5259fadeb710c5ac70d72955c3be55cf68e589584ee7f520c0b2a7bc8a20"
+  end
 
   def install
-    system "./bootstrap"
-    system "./configure", "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}"
-    system "make", "install"
+    ENV.libcxx if MacOS.version < :mavericks
```
Comment (DomT4) [inline Library/Formula/libcec.rb]: This can just be `ENV.libcxx`. No need to wrap it in OS X versions.

**Comment by DomT4** [comment]
Couple of nits but nearly ready to merge!

**Comment by nijikon** [comment]
@DomT4 please verify if any additional changes are needed, thanks!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/58037e251051ecd979afcc9a22288d415844ecd4. Thanks @nijikon!

**Pull request closed without merging**